tests/AnalyzeSpec.hs

```diff
@@ -1249,8 +1249,7 @@ spec = describe "analyze" $ do
                 it "should have no keyset provenance" $ do
                   ksProvs `shouldBe` Map.empty
 
-              other -> it "didn't find a single CheckFailure" $
-                HUnit.assertFailure $ show other
+              other -> runIO $ HUnit.assertFailure $ show other
```

Comment:
Why this change? Did HUnit change for GHC 8.4?

Reply:
Yes.